ts/att/time.ts

```diff
@@ -1,17 +1,70 @@
+import {
+    TimeSpan,
+} from "./timeSpan";
 export class Time {
 
     public static CreateTime24Hour(hour: number, minute: number): Time {
         return new Time(hour, minute);
     }
 
     public static CreateTime12Hour(hour: number, minute: number, isPm: boolean): Time {
-        if (hour < 1 || hour > 12) {throw new RangeError("Expected value is between 1 to 12, but actual is {hour}"); }
-        if (isPm && hour !== 12) {hour += 12; }
-        if (!isPm && hour === 12) {hour = 0; }
+        if (hour < 1 || hour > 12) {
+            throw new RangeError("Expected value is between 1 to 12, but actual is {hour}");
+        }
+        if (isPm && hour !== 12) {
+            hour += 12;
+        }
+        if (!isPm && hour === 12) {
+            hour = 0;
+        }
         const t = Time.CreateTime24Hour(hour, minute);
         return t;
     }
 
+    public static CreateTimeFrom12Hour(numbers: string, amOrpm: string): Time {
```

Comment:
What is this method for?

Reply:
You pass a string in 12-hour time format and pass a string indicating "am"/"pm" and it will create a 24-hour time